### PR TITLE
Add and Remove

### PR DESCRIPTION
**Pull request by Bantealem**
I have made add and remove website for to list

### CODE REVIEW

File: src/modules/actions.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  };
+
+  // edit a task in both the to-do tasks' array and  the localstorage
+  static editTask = (editId, userTask) => {
+    todoTasks[editId].description = userTask.value;
+    LocalStorage.set(todoTasks);
+    // Actions.displayTasks(todoTasks);
+    userTask.value = '';
+    return todoTasks;
+  };
```
Comment (vechicin) [inline src/modules/actions.js:82]: I cannot edit a task. Please fix this issue.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-plusplus */
```
Comment (vechicin) [inline src/index.js:1]: Please do not disable ESLint rules.

File: src/modules/actions.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-plusplus */
```
Comment (vechicin) [inline src/modules/actions.js:2]: Please do not disable ESLint rules.

File: src/modules/helpers.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-plusplus */
```
Comment (vechicin) [inline src/modules/helpers.js:1]: Please do not disable ESLint rules.

File: src/modules/taskStatus.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-plusplus */
```
Comment (vechicin) [inline src/modules/taskStatus.js:1]: Please do not disable ESLint rules.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Bantealem ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job implementing most of the features for this milestone

## Required Changes ♻️

- [ ] Be mindful of the way you are setting up linters. When I ran the `npx eslint .` locally, I got this error.
Screen Shot 2022-08-22 at 8 30 33

Please be mindful of **NOT EDITING** the config files if this is what caused the issue.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -1 +1,63 @@
-# my-to-do-list
\ No newline at end of file
+# Awesome Books
```
Comment (Bondok6) [inline README.md:1]: - [OPTIONAL] I suggest changing the name of the project to To Do List. 👌💯

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Bantealem 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job on implementing CRUD operations ✅
- Use ES6 modules to write modular JavaScript. ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I suggest adding the Link to your `License` file in your READM file 💯

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bantealem** [comment]
Bantealem/space-travelers-hub

**Pull request merged**